moped-editor/package.json

```diff
@@ -24,9 +24,6 @@
     "test": "react-scripts test",
     "eject": "react-scripts eject"
   },
-  "eslintConfig": {
-    "extends": "react-app"
-  },
```

Comment:
The `react-app`  ESLint config is Create React App's base lint configuration. Unless the ESLint behavior has changed, this config was previously being ignored, because we're using an `.eslintrc` file.

Docs:
```
If there are multiple configuration files in the same directory, ESLint will only use one.

The priority order is as follows:

.eslintrc.js
.eslintrc.cjs
.eslintrc.yaml
.eslintrc.yml
.eslintrc.json
package.json
```